packages/k/klayout/package.yml

```diff
@@ -17,6 +17,7 @@ builddeps  :
     - pkgconfig(Qt5XmlPatterns)
     - pkgconfig(python3)
     - pkgconfig(ruby-3.2)
+    - pkgconfig(libgit2)
```

Comment:
`pkgconfig` dependencies are ordered alphabetically with capitalized dependency names going first and then lower-case once going after those. So `pkgconfig(libgit2)` should be moved up and placed after `pkgconfig(Qt5XmlPatterns)` and before `pkgconfig(python3)`.